exercises/two-fer/src/example.erl

```diff
@@ -0,0 +1,13 @@
+-module(example).
+-export([two_fer/1, test_version/0]).
+
+two_fer(Name) ->
+  case Name of
+    "" ->
```

Comment:
I prefer to treat the empty string as a "name" given, and to return `"One for , one for me."`.